ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.cpp

```diff
@@ -149,7 +150,10 @@ void AnalyzeHotspots::processResults(GeoprocessingResult* result)
   connect(m_layer, &ArcGISMapImageLayer::doneLoading, this, [this](const Error& error)
   {
     if (error.isEmpty())
-        m_mapView->setViewpointGeometry(m_layer->fullExtent());
+    {
+      auto future = m_mapView->setViewpointGeometryAsync(m_layer->fullExtent());
+      Q_UNUSED(future);
```

Comment:
setViewpoint methods are no longer [[nodiscard]]. You shouldn't need Q_UNUSED or the future var anymore.

Reply:
My Windows machine code was old, it confused me. Sorry!, I will revert it.

Reply:
Done.